deploy/crds/druid.apache.org_druids_crd.yaml

```diff
@@ -1,43 +0,0 @@
-apiVersion: apiextensions.k8s.io/v1beta1
-kind: CustomResourceDefinition
-metadata:
-  name: druids.druid.apache.org
-spec:
-  group: druid.apache.org
-  names:
-    kind: Druid
-    listKind: DruidList
-    plural: druids
-    singular: druid
-  scope: Namespaced
-  subresources:
-    status: {}
-  validation:
-    openAPIV3Schema:
-      description: Druid is the Schema for the druids API
-      properties:
-        apiVersion:
-          description: 'APIVersion defines the versioned schema of this representation
-            of an object. Servers should convert recognized schemas to the latest
-            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
-          type: string
-        kind:
-          description: 'Kind is a string value representing the REST resource this
-            object represents. Servers may infer this from the endpoint the client
-            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
-          type: string
-        metadata:
-          type: object
-        spec:
-          description: DruidSpec defines the desired state of Druid
-          type: object
-        status:
-          description: DruidStatus defines the observed state of Druid
-          type: object
-      type: object
-  version: v1alpha1
-  versions:
-  - name: v1alpha1
-    served: true
-    storage: true
-    
```

Comment:
is this removing the CRD ?

Reply:
oh i see its in diff file, i wonder  where are we adding the kubebuilder tags for validation